src/webextension/list/manage/components/account-status.js

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import PropTypes from "prop-types";
+import React from "react";
+import { connect } from "react-redux";
```

Comment:
Redux stuff should go in the `containers/` directory. For really simple stuff (this might count unless you plan on adding more functionality to it), you can put everything in `containers`, but for more-complex stuff, you should create a purely presentational components in `components/` and then do the Redux hookup via `connect()` in `containers/`.

Reply:
At some point in the near future, clicking the button should open a dropdown menu for "Sign out" and "Preferences".

I can move this whole thing to `containers/` for now, and address that refactor when we get to it.